infoschema/tiniub/init.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tiniub
+
+import (
+	"sync"
+
+	"github.com/pingcap/parser"
+	"github.com/pingcap/parser/ast"
+	"github.com/pingcap/parser/model"
+	"github.com/pingcap/parser/mysql"
+	"github.com/pingcap/tidb/ddl"
+	"github.com/pingcap/tidb/expression"
+	"github.com/pingcap/tidb/infoschema"
+	"github.com/pingcap/tidb/meta/autoid"
+)
+
+var once sync.Once
+
+// Init initializes the TiNiuB database.
+func Init() {
+	initOnce := func() {
+		p := parser.New()
+		stmt, err := p.ParseOneStmt(tableSlowQuery, "", "")
+		if err != nil {
+			panic(err)
+		}
+		meta, err := ddl.BuildTableInfoFromAST(stmt.(*ast.CreateTableStmt))
+		if err != nil {
+			panic(err)
+		}
+		meta.ID = autoid.GenLocalSchemaID()
+		for _, c := range meta.Columns {
+			c.ID = autoid.GenLocalSchemaID()
+		}
+		dbInfo := &model.DBInfo{
+			ID:      autoid.GenLocalSchemaID(),
+			Name:    model.NewCIStr("TiNiuB"),
```

Comment:
Will the server fail to start if there is already a database which name is also 'TiNiuB'?
Maybe we should give some suggestions to the user when this happened: 1. disable this plugin and start again, 2. rename the existed database.
btw: it is better to use an English name, not a Chinese Pinyin as the database name.

Reply:
re, it's better give `tiniub` a English name? or can we direct put new table into `PERFORMANCE_SCHEMA` or `INFOMATION_SCHEMA`?

In mysql, we can write a `information_schema plugin` to  add a table to information_schema, https://dev.mysql.com/doc/refman/5.7/en/writing-information-schema-plugins.html